### DISCUSSION TITLE
Need ability to ERD of entire database and not only one schema at a time

### DISCUSSION THREAD

**Discussion by haisamido** [dbeaver/dbeaver#22364] [Q&A]
**Is your feature request related to a problem? Please describe.**
Need ability to ERD of entire Database and not only one schema at a time

**Describe the solution you'd like**
Select a database and then view its entire ERD for all the schemas in the database.

**Describe alternatives you've considered**
N/A

**Additional context**
database here is Postgres' definition, and schema is also Postgres's definition of a schema.

**Comment by LonwoLonwo** [comment]
Hello @haisamido 

Please follow these steps:

File -> New -> DBeaver -> ER Diagram -> Choose a name for a new diagram and your database.

Or you can find the Diagrams folder in your project and click on it to create a new diagram.

**Comment by haisamido** [comment]
> Hello @haisamido
> 
> Please follow these steps:
> 
> File -> New -> DBeaver -> ER Diagram -> Choose a name for a new diagram and your database.
> 
> Or you can find the Diagrams folder in your project and click on it to create a new diagram.

Thank you for your reply. That works, but curious why is it not readily available by right mouse clicking on the database as is the case for a schema?

**Comment by LonwoLonwo** [comment]
One of the reasons is potential misclicking. You can misclick on this tab in your databases tabs. Then, DBeaver can get stuck during metadata loading for the entire database.
So, database ERD creation should be intentional.
